constraint/pkg/client/drivers/k8scel/driver.go

```diff
@@ -0,0 +1,198 @@
+package k8scel
```

Comment:
you mentioned this is pending controller-runtime update?

Reply:
Correct. We can't use the K8s CEL code without the change that makes the result objects public (which is bleeding edge), and we can't use bleeding-edge k8s code until controller-runtime supports it.